source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,513 @@
+- **Device Sync Service (with Edge Server)**: use this option if you want to 
+  develop client apps using Atlas Device SDK and MongoDB Drivers and tools. 
+  This option involves more strict schema enforcement, which the SDKs use to 
+  automatically sync objects across devices and handle conflict resolution.
+
+Regardless of which type of service you choose, Edge Servers use Device Sync 
+to sync data between the Edge Server instance and Atlas.
+
+.. important:: You can't add Atlas Device SDK later to an Edge Server Service
+
+   You can always add Edge Servers to a Device Sync Service. But you can't use
+   Atlas Device SDK connections with an Edge Server Service. 
+   
+   Users are not portable between services. If you start with an Edge Server 
+   Service, but later want to develop apps using Atlas Device SDK, there is no 
+   way to transfer any users in your Edge Service to a new Device Sync Service.
+
+Before You Begin
+----------------
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      You need the following to manage Edge Servers in the Atlas UI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+   .. tab::
+      :tabid: api
+
+      You need the following to manage Edge Servers with the Admin API:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+      - .. include:: /includes/prereqs/atlas-admin-api-key.rst
+
+      - .. include:: /includes/prereqs/api-project-app-id.rst
+
+.. _create-edge-server-service:
+
+Create a Service for Edge Server
+--------------------------------
+
+You can create an Edge Server-compatible service in Data Services or App 
+Services. 
+
+- You might select Data Services when you just want to get started building
+  with Edge Server or Device Sync and Edge Server, and don't need other 
+  App Services. You can create an Edge Server Service from Data Services.
+- You might select App Services when you want to bundle your service with 
+  authentication, Functions, Triggers, HTTPS Endpoints, or Values and Secrets.
+  You can create a Device Sync Service from App Services, and add Edge Server
+  to it.
+
+.. tabs::
+
+   .. tab:: Create a Data Service
+      :tabid: atlas-service
+
+      .. include:: /includes/create-service-data-services.rst
+
+   .. tab:: Create an App Service
+      :tabid: app-service
+
+      Refer to the documentation to :ref:`create-app`.
+
+      When you choose your App type, select ``Device Sync``. You can add
+      Edge Server to your Device Sync App after you configure Device Sync.
+
+      [PLACEHOLDER: Add steps for adding Edge Server to a Device Sync App.
+      It's not 100% clear to me what the flow is from the Figma designs.
+      Populate the rest of this procedure when we are able to test this flow.]
+
+.. important:: Required Field
+
+   Documents that you want to sync with Edge Server must contain an ``_id`` 
+   field of type ``ObjectID``. Any documents without this field cannot sync 
+   to an Edge Server.
```

Comment:
This is only true if you are using the Edge Server service. For device sync, this is not required.

Reply:
We state [in the docs](https://www.mongodb.com/docs/atlas/app-services/sync/data-model/create-a-schema/#std-label-create-sync-schema) that Device Sync requires a primary key `_id` property, but I guess it doesn't have to be an `ObjectID` - our docs state it could also be a `string` or an `int`. So is that the difference - for Edge Server it has to be an `ObjectID` but for Device Sync, there must be an `_id` of a valid type? Or are the docs incorrect and `_id` isn't required for Sync?

Reply:
Device Sync Service:
 _id field can be of type ObjectId, String, Long, Int, UUID

Edge Server Service:
_id field has to be of type ObjectId

Reply:
I DM'ed as well and although the _id field actually isn't required for the edge server or device sync, we don't really encourage customers to do make it not-required so we can keep it in the docs that the _id field is required.